src/lib/utils/meta_tables/meta_system_information_table.cpp

```diff
@@ -51,9 +51,7 @@ size_t MetaSystemInformationTable::_cpu_count() {
     }
 
     cpu_info_file.close();
-  } catch (std::ios_base::failure& fail) {
-    Fail("Failed to read /proc/cpuinfo (" + fail.what() + ")");
-  }
+  } catch (std::ios_base::failure& fail) { Fail("Failed to read /proc/cpuinfo (" + fail.what() + ")"); }
```

Comment:
I dont see a benefit of pulling everything into one line. Was this done by accident?

Reply:
See above: might be a known problem with clang-format.